server/views/components/scroll-to-info/scroll-to-info.njk

```diff
@@ -0,0 +1,12 @@
+<a
+  href="#{{ model.elementId }}"
+  class="scroll-to-info js-scroll-to-info plain-link flex flex--column {{ 'scroll-to-info' | componentClasses(modifiers) if modifiers }} {% for class in data.extraClasses %}{{ class }} {% endfor %}"
+  data-track-click='{}'
+  data-component-name="Scroll to info"
+  data-component-id="{{ model.elementId }}"
+  data-speed="{{ model.speed }}"
+  data-easing="{{ model.easing }}"
+  data-element="{{ model.elementId }}">
```

Comment:
This really feels like it should be somewhere else?
Set it once, use it across the site?